tests/unit/test_alpaca_websocket.py

```
@@ -496,110 +496,84 @@ def test_unsubscribe_when_not_connected(self):
-    @pytest.mark.asyncio
-    async def test_reconnect_exponential_backoff(self):
-        """Test exponential backoff in reconnection."""
+    def test_reconnect_delay_calculation(self):
+        """Test exponential backoff delay calculation without async operations."""
         with (
             patch("core.data.alpaca_client.TradingClient"),
             patch("core.data.alpaca_client.StockHistoricalDataClient"),
-            patch("core.data.alpaca_client.StockDataStream"),
-            patch("asyncio.sleep", new_callable=AsyncMock) as mock_sleep,
         ):
             client = AlpacaClient(api_key="test_key", secret_key="test_secret")
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "secret_key"

(S106)